src/bg/updater.js

```diff
@@ -95,14 +95,13 @@ window._pickNextScriptAutoUpdate = async function() {
       return;
     }
 
-    let defaultCheckTime = new Date().getTime() + MIN_UPDATE_IN_MS;
     chrome.storage.local.get(updateNextAtKeys, vs => {
       for (let k of updateNextAtKeys) {
         let uuid = k.replace('updateNextAt.', '');
-        let v = vs[k];
+        let v = vs[k] || 0;
 
-        if (!nextTime || !nextUuid || v < nextTime) {
```

Comment:
Please keep code consistent with the existing style (keep `!nextTime`).

Reply:
@arantius You had said earlier that you wanted to use `let v = vs[k] || 0;`, so I took that to be condoning `nextTime !== null`. I'll have to do `let v = vs[k] || defaultCheckTime;` to actually solve the problem. Not sure if that will meet your style criteria, but I will prepare that variant of the change and restore the caveat of the 3 hour minimum browser runtime to the commit message.

Reply:
Err, even that doesn't work. I'll have to use a value of 1 there and check that inside of the `if`. This is getting way more complicated than removing the odd 3 hour pause behavior.